main.go

```diff
@@ -63,6 +32,7 @@ func main() {
     readEnv(&cfg)
     app.Initialize(
         readWIF(cfg.BlockChain.PrivateKeyPath), cfg.BlockChain.Url, cfg.BlockChain.ChainID,
-        cfg.Broker.TopicOffsetPath, cfg.Broker.Url, cfg.Broker.TopicID, cfg.Server.LogLevel)
+        cfg.Broker.TopicOffsetPath, cfg.Broker.Url, cfg.Broker.TopicID, cfg.BlockChain.CasinoAccountName,
```

Comment:
mb make struct like "AppCfg" with whole of this params?)

Reply:
yes, I'll try to fix that in upcoming commits